### ISSUE TITLE
CIN Con1d problems

### ISSUE THREAD

**Issue by EthanMaDev** [Leavingseason/xDeepFM#18]
In paper， section 3.2 CIN  analysis, 
> the number of weight at k-th layer is h_k * h_{k-1} *m.  
Howerver in you code, you use the con1d which has the weight of h_k * h_{k-1} *m * d. 

should we use the  tf.nn.depthwise_conv1d but the con1d, hope your responses

**Comment by Leavingseason** [comment]
I couldn't find the function of 'depthwise_conv1d'.  Considering that conv1d is actually implemented via reshaping the input and then send it to conv2d, may be what you mentioned is exactly the current implementation of conv1d?

**Comment by EthanMaDev** [comment]
In your version of CIN， the number of weight in k-th layer is field_nums[-1] * field_nums[0] *layer_size * dim。
The Con1d is the two-dimensional weight. But in the CIN analysis(section 3.2) , the CIN is independant of embedding dim which is just ( h_k * h_{k-1) * m  in k-th layer)
